api/webhook.go

```diff
@@ -139,7 +139,7 @@ func PostWebhook(c *gin.Context) {
 	}()
 
 	// check if build was parsed from webhook
-	if b == nil {
+	if b == nil && h.GetEvent() != "repositoryRename" {
```

Comment:
Should this reference the `EventRepositoryRename` constant?

https://github.com/go-vela/types/blob/master/constants/event.go#L24-L25